docs/install/Knative-custom-install.md

```diff
@@ -173,6 +173,8 @@ The following Knative installation files are available:
   - https://github.com/knative/serving/releases/download/v0.5.0/monitoring-metrics-prometheus.yaml
   - https://github.com/knative/serving/releases/download/v0.5.0/monitoring-tracing-zipkin.yaml
   - https://github.com/knative/serving/releases/download/v0.5.0/monitoring-tracing-zipkin-in-mem.yaml
+  - https://github.com/knative/serving/releases/download/v0.5.0/monitoring-tracing-jaeger.yaml
```

Comment:
```suggestion
  - https://github.com/knative/serving/releases/download/v0.5.0/monitoring-tracing-zipkin.yaml
```

Reply:
I am really confused by this PR. The title is about adding Jaeger tracing manifests. But here is a suggestion to switch Jaeger to Zipking (two different tracing software).

So either the title of the PR is wrong, or this suggestion is off.

Finally, if the goal is really to add a reference to the Jaeger manifests, these *do not* exist and are not part of the releases (at least yet). See: https://github.com/knative/serving/releases

Reply:
The suggestion was related to changing the ordering of the links, so putting the jaeger links above zipkin, so in alphabetical order. I had applied the changes separately - but looks like the commit suggestions are still showing for some reason - but if you look at the current state of the file you can see the links are in correct order.

The jaeger manifests will be created during the next release - as part of merged PR https://github.com/knative/serving/pull/3696.

Reply:
ok, so any reason why the version is `v0.5.0` and not `v0.6.0` . Since the manifests will never be available at the v0.5.0 path ?

Reply:
As discussed https://github.com/knative/docs/pull/1154#discussion_r274070392, it was agreed to leave as `0.5.0` as all of the links are updated when the next release is done. If its a blocker then I can update them?

Reply:
so if it gets merged before 0.6.0 release then the links will 404 .

Reply:
if the links are updated to `0.6.0` the same thing would happen - so the only solution is to merge the PR when the release is being done.

Reply:
imo, it would be less confusing to have this PR already list the `0.6.0` release, label it for `0.6.0` docs release and merge it on release day. Anyway, looks like we agree. thanks for clarifying.

Reply:
Links updated.